### ISSUE TITLE
Add NW?

### ISSUE THREAD

**Issue by armintoepfer** [mgawan/GPU-BSW#16]
Hi!

Is it possible to change the alignment kernel to compute the Needleman-Wunsch alignment? It's basically starting at the last entry and not the max score.

Thanks,
Armin

**Comment by mgawan** [comment]
yes, that should be possible. I will look into this and post here once I have the implementation.
